lib/configgin.rb

```diff
@@ -66,6 +87,51 @@ def render_job_templates(jobs, job_configs)
     end
   end
 
+  # Some pods might have depended onthe properties exported by this pod; locate
+  # them and cause them to restart as appropriate.
+  def restart_affected_pods(job_configs, job_digests)
+    raise 'No digest' unless job_digests
+
+    instance_groups_to_examine = Hash.new { |h, k| h[k] = {} }
+    job_configs.values.each do |job_config|
+      base_config = JSON.parse(File.read(job_config['base']))
+      base_config['consumed_by'].each_pair do |provider_name, consumer_jobs|
+        consumer_jobs.each do |consumer_job|
+          digest_key = "skiff-imported-properties-#{instance_group}-#{provider_name}"
+          instance_groups_to_examine[consumer_job['role']][digest_key] = job_digests[provider_name]
+        end
+      end
+    end
+
+    instance_groups_to_examine.each_pair do |instance_group_name, digests|
+      begin
+        kube_client_stateful_set.patch_stateful_set(
+          instance_group_name,
+          { spec: { template: { metadata: { annotations: digests } } } },
+          kube_namespace
+        )
+        warn "Patched StatefulSet #{instance_group_name} for new exported digests"
+      rescue KubeException => e
+        begin
+          # rubocop:disable Layout/RescueEnsureAlignment
+          response = begin
+            JSON.parse(e.response || '') || {}
+          rescue JSON::ParseError
+            {}
+          end
+          # rubocop:enable Layout/RescueEnsureAlignment
+          if response['reason'] == 'NotFound'
+            # The StatefulSet can be missing if we're configured to not have an optional instance group
```

Comment:
[nit] Punctuation.

Reply:
Fixed in da315a15408eec9730f51d8d295c513d19f22eb2